utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/conversion.scala

```diff
@@ -138,6 +139,30 @@ trait ConversionUtils extends HideToString {
     case _                       => null
   }
 
+  @Documentation(description = "Convert String value to JSON")
+  def toJson(@ParamName("value") value: String): Any = {
+    toJsonEither(value).toTry.get
```

Comment:
I don't like that we return here Circe.. It can be a problematic type for Flink serialization / deserialization :/ On ohter hand, what we can return here? Maybe we can try convert Json to java types?